sources/europe/it/SouthTyrol-Cadastre.geojson

```diff
@@ -0,0 +1,934 @@
+{
+    "type": "Feature",
+    "properties": {
+        "attribution": {
+            "url": "https://geoservices.buergernetz.bz.it/geokatalog",
+            "text": "© Autonome Provinz Bozen/Provincia Autonoma di Bolzano CC-BY 4.0",
+            "required": false
+        },
+        "license_url": "https://geoportal.buergernetz.bz.it/veroeffentlichungen.asp?publ_action=300&publ_image_id=394227",
+        "privacy_policy_url": "http://geoportal.buergernetz.bz.it/privacy.asp",
+        "url": "https://geoservices5.civis.bz.it/geoserver/p_bz-Cadastre/ows?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=ParcelsPolygons&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+        "max_zoom": 20,
+        "country_code": "IT",
+        "available_projections": [
+            "EPSG:3857",
+            "EPSG:25832",
+            "EPSG:32632"
+        ],
+        "type": "wms",
+        "category": "photo",
+        "id": "SouthTyrol-Cadastre",
+        "name": "South Tyrol Cadastre",
+        "start_date": "2024-09-02",
+        "end_date": "2024-09-02",
```

Comment:
If the data is continually updated, best leave off the start/end dates. Unless you know the data is fixed at this time period?